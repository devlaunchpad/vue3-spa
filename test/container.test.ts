import { mount } from '@vue/test-utils'
import { describe, expect, it } from 'vitest'
import Container from '../src/components/Container.vue'

describe('component Container.vue', () => {
  it('should render correctly', () => {
    const wrapper = mount(Container)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
