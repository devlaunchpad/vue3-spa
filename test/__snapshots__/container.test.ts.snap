// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`component Container.vue > should render correctly 1`] = `"<div class="container"></div>"`;
